src/btree/bt_curnext.c

```diff
@@ -730,7 +741,7 @@ __wt_btcur_next_prefix(WT_CURSOR_BTREE *cbt, WT_ITEM *prefix, bool truncating)
                 continue;
             }
         }
-
+skip_page:
```

Comment:
Should "skip_page" goto label jump to line 766 instead, i.e. to:
` WT_ERR(__wt_tree_walk(session, &cbt->ref, flags));`

Why check for deleted records on the page we skipped reading, they will always be 0. Am I missing something ?

Reply:
Fixed.